antarest/launcher/service.py

```diff
@@ -0,0 +1,51 @@
+from uuid import UUID
+
+from antarest.common.config import Config
+from antarest.launcher.factory_launcher import FactoryLauncher
+from antarest.launcher.model import JobResult, JobStatus
+from antarest.launcher.repository import JobResultRepository
+from antarest.storage.service import StorageService
+
+
+class JobNotFound(Exception):
+    pass
+
+
+class LauncherService:
+    def __init__(
+        self,
+        config: Config,
+        storage_service: StorageService,
+        repository: JobResultRepository,
+        factory_launcher: FactoryLauncher = FactoryLauncher(),
+    ) -> None:
+        self.config = config
+        self.storage_service = storage_service
+        self.repository = repository
+        self.launcher = factory_launcher.build_launcher(config)
+
```

Comment:
add a threaded loop to watch for running job status by polling run_study.
sleep interval should be configurable (default 5s may be ok)
it will poll the launcher for study status and remove the watched study when completion occur or a configurable timeout (default 7 days). the status will be updated if different from the in memory cache list of running jobs.